repos/os/src/drivers/platform/spec/x86/pci_config_access.h

```diff
@@ -100,26 +89,34 @@ namespace Platform {
 			              unsigned char addr, Device::Access_size size,
 			              bool track = true)
 			{
-				/* write target address */
-				_io_port<REG_ADDR>()->outl(REG_ADDR, _cfg_addr(bus, device, function, addr));
+				unsigned ret;
+				const addr_t mmconf = _base + _dev_base(bus, device, function);
```

Comment:
We prefer to post-fix cv-qualifiers (const, volatile) to the type, so this line should start with `addr_t const mmconf...`

Reply:
Will do.